mu-plugins/central-hub/tests/phpunit/integration/meta-data/isOkayToSaveMetaBox.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Tests for the function is_okay_to_save_meta_box().
+ *
+ * @package     spiralWebDb\centralHub\Tests\Integration\Metadata
+ * @since       1.3.0
+ * @author      Robert Gadon <rgadon107>
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Integration\Metadata;
+
+use function spiralWebDB\Metadata\is_okay_to_save_meta_box;
+use function wp_create_nonce;
+use function wp_verify_nonce;
+use function did_action;
+use spiralWebDb\Cornerstone\Tests\Integration\Test_Case;
+
+/**
+ * Class Tests_IsOkayToSaveMetaBox
+ *
+ * @package spiralWebDb\centralHub\Tests\Integration\Metadata
+ * @group   meta-data
+ */
+class Tests_IsOkayToSaveMetaBox extends Test_Case {
+
+	/**
+	 * Set up the test environment before each test.
+	 */
+	public function setUp() {
+		parent::setUp();
+		$_POST = [];
+	}
+
+	/**
+	 * Clean up the test environment after each test.
+	 */
+	public function tearDown() {
+		parent::tearDown();
+	}
+
+	/**
+	 * Test is_okay_to_save_meta_box() should check the meta box key exists in the $_POST array.
+	 */
+	public function test_should_return_false_when_meta_box_key_is_not_in_POST() {
+		$_POST            = [
+			'post_ID'           => '1322',
+			'post_status'       => 'publish',
+			'events'            => [
+				'event-date' => '09-24-2019',
+				'event-time' => '17:30',
+				'venue-name' => 'Ladue Chapel',
+			],
+			'events_nonce_name' => wp_create_nonce( 'events_nonce_action' ),
+		];
+
+		$this->assertFalse( array_key_exists( 'members', $_POST ) );
```

Comment:
@rgadon107 Please use the `assertArrayNotHasKey` for this one.

Reply:
Done.